### PR TITLE
Bug 1836844 - Add esr115 repos

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

File: config2.json [hunk 1]
```diff
@@ -34,6 +34,35 @@
...
+      "cache": "codesearch",
+      "index_path": "$WORKING/mozilla-esr115",
+      "files_path": "$WORKING/mozilla-esr115/git",
+      "git_path": "$WORKING/mozilla-esr115/git",
+      "git_blame_path": "$WORKING/mozilla-esr115/blame",
+      "github_repo": "https://github.com/mozilla/gecko-dev",
+      "hg_root": "https://hg.mozilla.org/releases/mozilla-esr115",
+      "objdir_path": "$WORKING/mozilla-esr115/objdir",
+      "codesearch_path": "$WORKING/mozilla-esr115/livegrep.idx",
+      "codesearch_port": 8083
```
Comment (jcristau) [inline config2.json:49]: should this use a different port than esr102?

Comment (staktrace) [reply config2.json:49]: Yes!

**Comment by staktrace** [comment]
I'm going to do some reshuffling here. esr91 can move to config3 since it's not getting any new commits, etc.

File: config2.json (hunk 1 above)
Comment (staktrace) [reply config2.json]: Fixed this

**Comment by staktrace** [comment]
I'm generating the necessary comm-esr115 tarballs and uploading them. Will also need to wait for the currently running config1 indexer to finish before merging this.

**Comment by staktrace** [comment]
Tarballs uploaded, the config1 is far enough along that it's uploaded new gecko.tar and gecko-blame.tar files, and I've updated the load balancer rules to account for the reshuffle. Merging this now

**Comment by emilio** [comment]
Thanks for this Kats! <3

For context: I had seen this (though late b/c my github notifications are a mess). I had pending figuring out how to do the comm-central tarball dance but then I got swamped with other work and PTO, so it was going to have to wait until at least next week when I'm back. Anyways, thanks a lot again for that and the follow-ups!

**Comment by asutherland** [comment]
Yes, many, many thanks to @staktrace for the review and related deployment and AWS work and cluster of fixes cleaning up the long tail of surprises!  Also thank you to @hneiva for the initial patch!

**Comment by hneiva** [comment]
Thanks ya'll for the effort landing this! 🚀 🤝

**Pull request merged**